jest.config.js

```diff
@@ -3,5 +3,5 @@ module.exports = {
   transform: {
     '^.+\\.tsx?$': 'ts-jest',
   },
-  testMatch: ['**/?(*.)+(spec|test).ts'],
+  testMatch: ['**/?(*.)+(spec|test|integ).ts'],
```

Comment:
Out of curiosity, is `integ` the new way folks call `e2e`?
I am not agaist that, I am just curios as I never saw this prefix before.
cc @robpolak @cnasc